source/manual/get-started.html.md

```diff
@@ -12,273 +12,25 @@ This is the guide for new technical staff working on GOV.UK in [GDS][]. If you j
-1. Import the SSH key into your keychain. Once you’ve done this, it’ll be available to the VM you'll install in the next step.
-
-        mac$ /usr/bin/ssh-add -K your-private-key
-
-1. Test that it all works by running `ssh -T git@github.com`.
-
-1. While you're here, associate your name and email to your git commits:
-
-    mac$ git config --global user.email "friendly.giraffe@digital.cabinet-office.gov.uk"
-    mac$ git config --global user.name "Friendly Giraffe"
-
-[GitHub]: https://www.github.com/
-[user-reviewer]: https://github.com/alphagov/govuk-user-reviewer
-[alphagov]: https://github.com/alphagov
-[govuk-team]: https://github.com/orgs/alphagov/teams/gov-uk/members
-[register-ssh-key]: https://help.github.com/articles/connecting-to-github-with-ssh/
-
-## 3. Set up your local workspace
-
-Some processes are dependent on your local gov.uk setup being in `~/govuk`:
-
-    mac$ mkdir ~/govuk
-    mac$ cd ~/govuk
-    mac$ git clone git@github.com:alphagov/govuk-puppet.git
-
-### 3.1 Ensure your Ruby setup works
-
-You'll want to install the [specific version of Ruby in govuk-puppet](https://github.com/alphagov/govuk-puppet/blob/master/.ruby-version#L1) (currently `rbenv install 1.9.3-p550`), as the default system Ruby is too locked-down.
-
-Now verify that rbenv is working as expected:
-
-    mac$ cd ~/govuk/govuk-puppet
-    mac$ rbenv versions
-
-This should output something like:
-
-```
-  system
-* 1.9.3-p550 (set by /Users/yourname/govuk/govuk-puppet/.ruby-version)
-```
-
-    mac$ which ruby
-
-This should output something like:
-
-```
-/Users/yourname/.rbenv/shims/ruby
-```
-
-If, instead, this outputs `/usr/bin/ruby`, then you'll need to update your `~/.bash_profile` to have `ruby` properly overridden in your `PATH`:
-
-```bash
-# force load rbenv to ensure correct version of ruby used
-export PATH="$PATH:~/.rbenv/shims"
-if which rbenv > /dev/null; then eval "$(rbenv init -)"; fi
-```
-
-This Ruby setup is a pre-requisite for the "[Import production data](#7-import-production-data)" step.
-
-## 4. Boot your VM
+[overview-slides]: https://docs.google.com/presentation/d/1nAE65Og04JYNAc0VjYaUYLqNLuUOM9r3Mvo0PGFy_Zk
 
-Run the VM bootstrap script:
+## 1. Set up your GitHub account
 
-    mac$ cd ~/govuk/govuk-puppet/development-vm
-    mac$ vagrant up
-    mac$ vagrant dns --install
+👉 [Set up your GitHub account](/manual/github-setup.html)
 
-This will take a little while, but it will throw up a question or two in your console so check back on it occasionally. Now might be a good time to scan through the [GOV.UK technology blog][govuk-tech-blog] while Puppet runs.
+## 2. Set up govuk-docker
 
-Once your VM is running, you can SSH into it with:
+The easiest way to develop on GOV.UK is to use [govuk-docker][]. Until July 2019 we used a Vagrant development VM, but this is now deprecated. You can [still find the installation instructions in this guide](/manual/install-development-vm.html).
```

Comment:
```suggestion
The easiest way to develop on GOV.UK is to use [govuk-docker][]. Until July 2019 we used a [GOV.UK Vagrant development VM](/manual/install-development-vm.html), but this is now deprecated.
```

(just making the link standalone so that it makes sense to screen reader users)